.rspec

```diff
@@ -1 +1,2 @@
 --require spec_helper
+--format documentation
```

Comment:
think it might be better to actually un-comment the additional section in the spec_helper.rb https://github.com/Luke-kb/mySMS/blob/develop/spec/spec_helper.rb#L49.
It actually has a piece in there that will change the format to documentation if you run one test.

```ruby
  # Many RSpec users commonly either run the entire suite or an individual
  # file, and it's useful to allow more verbose output when running an
  # individual spec file.
  if config.files_to_run.one?
    # Use the documentation formatter for detailed output,
    # unless a formatter has already been configured
    # (e.g. via a command-line flag).
    config.default_formatter = "doc"
  end
```

Generally this will need to happen in a separate PR but happy to make an exception for this 👍🏽